tests/test_simple_shear_2d.py

```diff
@@ -475,7 +480,15 @@ def test_dvdx_GBM(self, outdir, seeds_nearX45, ncpus):
                         "Zhang & Karato, 1995\n(1300°C)",
                     ],
                 )
-                fig.savefig(_io.resolve_path(f"{out_basepath}.pdf"))
+                # There is a lot of stuff on this legend, so put it outside the axes.
+                # The 'loc' argument doesn't seem to work, use `bbox_to_anchor` instead.
```

Comment:
`loc` and `bbox_to_anchor` work together: `loc` sets the location of the legend box that becomes the target of `bbox_to_anchor`.

Reply:
I see, I'll remove that line since it works fine with the default `loc`.